src/utils/src/ExponentialBackoffRetryStrategy.c

```diff
@@ -232,6 +232,20 @@ STATUS getExponentialBackoffRetryStrategyWaitTime(PRetryStrategy pRetryStrategy,
     return retStatus;
 }
 
+STATUS getExponentialBackoffRetryCount(PRetryStrategy pRetryStrategy, PUINT32 retryCount) {
```

Comment:
How/where will this be used?  The question is really do we need to pass a `PRetryStrategy` to this function it looks like it doesn't actually use anything from it except for the retry state, can we just pass it the `PRetryState` instead?  Perhaps there is a good reason to have it, would like to understand.

Reply:
PRetryState is supposed to be an internal structure. This PR moves the struct to `Include_i.h`